ui/app/components/ui/popover/index.scss

```diff
@@ -0,0 +1,234 @@
+.popover {
+  &-content {
+    ::-webkit-scrollbar {
+      width: 6px;
+    }
+
+    ::-webkit-scrollbar-thumb {
+      -webkit-border-radius: 10px;
+      border-radius: 10px;
+      background: #C4C4C4;
+    }
+
+    position: relative;
+    width: 328px;
+    height: 564px;
+
+    box-shadow: 0px 4px 30px rgba(0, 0, 0, 0.25);
+    border-radius: 10px;
+    background: white;
+    z-index: 2;
+    left: calc(50% - 328px/2);
+    top: 10px;
+
+    .react-tabs {
+      height: calc(100% - 66px);
+    }
+
+    .react-tabs__tab-panel--selected {
+      height: 100%;
+    }
+
+    .react-tabs__tab-list {
+      display: flex;
+      justify-content: flex-start;
+      align-items: stretch;
+      flex-direction: row;
+    }
+
+    .react-tabs__tab {
+      flex-grow: 1;
+      text-align: center;
+      font-weight: 500;
+      font-size: 14px;
+      line-height: 140%;
+      display: block;
+      padding: 14px 0;
+      color: #6A737D;
+      border-bottom: 1px solid #C4C4C4;
+    }
+
+    .react-tabs__tab--selected {
+      color: $lochmara;
+      border-color: $lochmara;
+      border-width: 2px;
+    }
+  }
+
+  &-item {
+    display: flex;
+    justify-content: flex-start;
+    align-items: stretch;
+    border-top: 1px solid $mercury;
+    padding: 10px 0;
+
+    &:first-child {
+      border-top: none;
+    }
+
+    &__col {
+      align-self: flex-start;
+      &:last-child {
+        margin: 0 0 0 auto;
+        text-align: right;
+      }
+    }
+
+    &__heading {
+      font-size: 10px;
+      line-height: 180%;
+      text-transform: uppercase;
+      color: $Grey-500;
+    }
+
+    &__subheading {
+      font-size: 12px;
+      line-height: 160%;
+      color: #24292E;
+      &.confirmed {
+        color: #28A745;
+      }
+    }
+    &__icon {
+      padding: 4px;
+    }
+  }
+
+  &-header {
+    &__heading {
+      @extend %font;
+      font-weight: bold;
+      font-size: 18px;
+      line-height: 25px;
+      text-align: center;
+      padding: 20px 0;
+      border-bottom: 1px solid #DDDEE9;
+    }
+    &__close {
+      position: absolute;
+      top: 13px;
+      right: 6px;
+      width: 18px;
+      height: 18px;
+      display: block;
+      padding: 20px;
+
+      &--wrap {
+        width: 18px;
+        height: 18px;
+        transform: rotate(45deg);
+        pointer-events: none;
+        position: absolute;
+        top: 10px;
+        left: 10px;
+      }
+
+      &--line {
+        &-a, &-b {
+          width: 18px;
+          height: 2px;
+          background: #4A4A4A;
+          position: absolute;
+          border-radius: 2px;
+          top: calc(18px / 2);
+        }
+
+        &-b {
+          transform: rotate(90deg);
+        }
+      }
+    }
+  }
+
+  &-bg {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: black;
+    opacity: 20%;
+    z-index: 1;
+  }
+
+  &__advanced-input-controls {
```

Comment:
A lot of these styles appear unrelated to the popover component

Reply:
These styles still appear to be present (the popup doesn't have any `advanced-input-controls`)